### DISCUSSION TITLE
Support for inspecting elements with VS Live Visual Tree and editing properties

### DISCUSSION THREAD

**Discussion by h0lg** [CommunityToolkit/Maui.Markup#306] [Q&A]
I notice I can't use Visual Studio's _Live Visual Tree_ to inspect into views built using C# markup extensions. That significantly increases the number of times I have to rebuild the app in the layout/design loop because I can't fiddle with the visual properties in the live property editor until I have found a setting that I like.

Will these features ever be supported or is this the drawback of not using XAML views?

**Comment by TheCodeTraveler** [comment]
The docs suggest enabling Hot Reload and disabling **Just My XAML**:

> Provided that Hot Reload is enabled, the Live Visual Tree window will display the hierarchy of your app's UI elements regardless of whether the app's UI is built using XAML or C#. However, you will have to disable Just My XAML to display the hierarchy of your app's UI elements for UIs built using C#.


https://learn.microsoft.com/dotnet/maui/user-interface/live-visual-tree
